presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/ImplementBernoulliSampleAsFilter.java

```diff
@@ -60,7 +66,9 @@ public Result apply(SampleNode sample, Captures captures, Context context)
                 sample.getSource(),
                 new ComparisonExpression(
                         ComparisonExpression.Operator.LESS_THAN,
-                        new FunctionCall(QualifiedName.of("rand"), ImmutableList.of()),
+                        new FunctionCallBuilder(metadata)
```

Comment:
These calls for zero argument functions seem quite verbose. Maybe add a simple factory method
```java
buildFunctionCall(metadata, "rand");
```